### DISCUSSION TITLE
articles/rms

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#148] [Comments]
http://hutusi.com/articles/rms

 在开源界有很多关于Richard M.Stallman（以下简称RMS）的传闻，比如说：

**Comment by HarryWang2017** [comment]
free software和opensource software，确实让人迷惑。
我理解，自由应该是人类追求的终极目标，然而可能永远也达不到，只能不断逼近。
折射在软件领域，自由软件是终极目标。在逼近这一目标过程中，有了开源软件。
完全的自由软件是不可能的，因为不是人人都是程序员。总有人是需要被别人服务，而服务是有偿的。软件即服务。
开源软件，其实是降低了提供服务人员的进入门槛，避免因为有人持有软件而获得垄断地位，获得垄断收益。

**Comment by hutusi** [comment]
@HarryWang2017 谢谢你的评论。我觉得是否有偿服务并不是阻碍自由软件发展的问题，因为用 RMS 的话说，free software 是 free speech 的 free, 而不是 free beer 的 free. 自由软件不是免费软件，收取服务甚至出售软件都是可以的。自由软件联盟过于关注软件的自由，反倒束缚了手脚，也忽视了商业公司的利益，以至于人们更多的谈论开源而非自由了。
